packages/asc-ui/src/components/Link/Link.stories.tsx

```diff
@@ -2,8 +2,30 @@ import * as React from 'react'
 import { storiesOf } from '@storybook/react'
 import Link from './Link'
 
-storiesOf('Atoms/Typography/Link', module).add('default style', () => (
-  <>
-    <Link href="/">I am a link!</Link>
-  </>
-))
+storiesOf('Atoms/Typography/Link', module)
+  .addDecorator(storyFn => (
+    <div style={{ padding: '40px 10px', background: 'rgb(248,248,248,0.8)' }}>
+      {storyFn()}
+    </div>
+  ))
+  .add('default type (link with chevron icon)', () => (
```

Comment:
And this "default type with chevron icon" -> would switch the two stories

Reply:
already done that :-)

Reply:
done